java-generator/core/pom.xml

```diff
@@ -68,9 +68,8 @@
     </dependency>
 
     <dependency>
-      <groupId>javax.validation</groupId>
```

Comment:
Nice! would you mind removing the dependency from the integration tests(and everwhere under the `java-generator` folder) and in the relevant docs?

Reply:
Nice catch, thanks. Will do.